packages/kbn-test/src/functional_test_runner/lib/config/ftr_configs_manifest.ts

```diff
@@ -26,6 +27,32 @@ interface FtrConfigsManifest {
   enabled: Array<string | FtrConfigWithOptions>;
 }
 
+// eslint-disable-next-line no-console
+console.log(`REPO_ROOT: ${REPO_ROOT}`);
+// eslint-disable-next-line no-console
+console.log(`cwd: ${process.cwd()}`);
+
+ls('.');
+ls(REPO_ROOT);
+ls(`${REPO_ROOT}/.buildkite`);
+ls(
```

Comment:
am I crazy, shouldn't these `ls()` calls be awaited?  I know it's orthogonal to this ci issue we are working on but wuttt?? lol

Reply:
Nvm, I just tried this in a spike file, worked fine. 😱